indexer/selector.go

```diff
@@ -53,7 +56,7 @@ func (s *selectorBlock) Text(el *goquery.Selection) (string, error) {
 			WithFields(logrus.Fields{"case": s.Case}).
 			Debugf("Applying case to selection")
 		for pattern, value := range s.Case {
-			if el.Is(pattern) {
+			if el.Is(pattern) || el.Has(pattern).Length() >= 1 {
```

Comment:
Is requires the root element of the el selection to match the selector (pattern). Has() matches for the child elements.